src/UI/Pages/Agents/Actions.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useContext, useState } from "react";
+import {
+  Button,
+  Dropdown,
+  DropdownItem,
+  KebabToggle,
+} from "@patternfly/react-core";
+import { DependencyContext, words } from "@/UI";
+
+interface Props {
+  name: string;
+  paused: boolean;
+}
+
+export const Actions: React.FC<Props> = ({ name, paused }) => {
+  const { commandResolver } = useContext(DependencyContext);
+  const deploy = commandResolver.getTrigger<"Deploy">({ kind: "Deploy" });
+  const repair = commandResolver.getTrigger<"Repair">({ kind: "Repair" });
+  const [isOpen, setIsOpen] = useState(false);
+  return (
+    <>
+      <Button variant="secondary" isDisabled isSmall>
```

Comment:
I assume this action will be added in the future?

Reply:
Indeed